core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.service.ts

```diff
@@ -77,6 +77,14 @@ export class TopicsAndSkillsDashboardPageService {
         if (filterObject.classroom === 'Unassigned' && !topic.classroom) {
           return true;
         }
+
+        if (
```

Comment:
This function should not return objects, it returns booleans, but you return topic.classroom

Reply:
the classroom is of type null or string. so it just check whether the topic is assigned to some classroom or not.

Reply:
Let's return !!topic.classroom then?

Reply:
yes sure, returning boolean makes more sense.